Lesson07/HW07.StringWorkerThree/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+
+namespace HW07.StringWorkerThree
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {          
+            string initialВata = "gdfgdf234dg54gf+23oP42";
+            Console.WriteLine("Изначальное значение: " + initialВata);
+            string[] signSeparation = initialВata.Split('+','-','*','/');
+
+            foreach (var word in signSeparation)
+            { }                      
```

Comment:
What does this cycle do? For what is it here?

Reply:
Done